src/actions/go-to.ts

```diff
@@ -0,0 +1,22 @@
+import 'colors';
+import { Browser } from 'puppeteer';
+import Action from '../action.i';
+
+/**
+ * Opens a new window for a given browser and navigates to a specified url.
+ */
+
+export default class GoTo implements Action {
+  constructor() {}
+
+  // @ts-ignore
```

Comment:
This will disable all type checks. In most cases, type errors indicate a mistake on our end. Please try to fix the error.

Reply:
Done